ee/clickhouse/queries/funnels/test/test_funnel.py

```diff
@@ -1538,3 +1538,24 @@ def test_funnel_with_groups_global_filtering(self):
 
         self.assertEqual(result[1]["name"], "paid")
         self.assertEqual(result[1]["count"], 0)
+
+    def test_breakdown_values_is_set_on_the_query_with_fewer_than_two_entities(self):
+
+        filter_with_breakdown = {
+            "events": [{"id": "user signed up", "type": "events", "order": 0},],
+            "insight": INSIGHT_FUNNELS,
+            "date_from": "2020-01-01",
+            "date_to": "2020-01-14",
+            "breakdown": "something",
+            "breakdown_type": "event",
+        }
+        filter = Filter(data=filter_with_breakdown)
+        funnel = ClickhouseFunnel(filter, self.team)
+
+        # because this calls get_step_counts_query
+        # which calls get_step_counts_without_aggregation_query
+        # which calls _get_inner_event_query
+        # which calls _get_breakdown_conditions
+        funnel.get_query()
```

Comment:
non-blocking

Does it fail if you call `.run()`? I'm thinking worth not making the test rely on `get_query` and `params`  but rather probably the more stable interface. What do you think?